lib/components/LandingPage.jsx

```diff
@@ -7,15 +7,15 @@ import {
   PageTitleAndBreadcrumbs
 } from '@pooltogether/react-components'
 import Link from 'next/link'
-import classnames from 'classnames'
 import React from 'react'
 import { Trans, useTranslation } from 'react-i18next'
 
-import ChatBubble from 'assets/images/LandingPage/chat-bubble.png'
-import DollarBill from 'assets/images/LandingPage/dollar-bill.png'
-import VoteIcon from 'assets/images/LandingPage/vote-icon.png'
 import { DISCORD_INVITE_URL, POOLTOGETHER_GOV_FORUM_URL } from 'lib/constants'
 
+import ChatBubble from 'assets/images/chat-bubble@2x.png'
```

Comment:
We should start using just flattened svg's with fills. They scale and look significantly better & we could customize them (set a gradient fill, purple, teal, hover effects etc). Pngs are always so pixelated.

The gradients on these look funny - the dots in the chat bubble & dollar bill don't fit the rest of the image.